platform/solr/solr-factory-impl/src/main/java/org/codice/solr/factory/impl/SolrCloudClientFactory.java

```diff
@@ -117,11 +118,23 @@ RetryPolicy withRetry() {
   public void createCollection(String collection, CloudSolrClient client)
       throws SolrFactoryException {
     try {
+      CollectionAdminResponse aliasResponse =
```

Comment:
:question 
Should alias have a configuration? Should this check be done for `uploadCoreConfiguration` as well.

Reply:
It will be ok to upload configurations even if the name is for an alias. I could see us having a "catalog" configuration that supports multiple "catalog" collections (e.g. catalog_prod, catalog_test, etc)